app/assess/forms/resolve_flag_form.py

```diff
@@ -0,0 +1,14 @@
+from flask_wtf import FlaskForm
+from wtforms import TextAreaField, RadioField
+from wtforms.validators import DataRequired
+
+class ResolveFlagForm(FlaskForm):
+    resolution_flag = RadioField(
+        "resolve", 
+        choices=["RESOLVED", "STOPPED"], 
+        validators=[DataRequired()]
+    )
+    justification = TextAreaField(
+        "justification",
+        validators=[DataRequired()],
```

Comment:
For text area can we use `InputRequired`?

Reply:
good spot 👍 
context: Certain field types coerce data into Falsey values(0, Decimal(0), etc.). The problem occurs when you have an IntegerField and the form submits a value like '0'. If you apply DataRequired to this it will fail validation. This is because DataRequired will evaluate if not field.data... after coercion where field.data is the Falsey numeric value 0.

Reply:
With this in mind, I could change the other data_required validators to input_required as part of this PR?

Reply:
Yes, maybe I think the recommendation is it to use this instead of `DataRequired`

Reply:
cool - ive rolled out to other forms 👍